force-app/main/default/classes/USER_UserService.cls

```diff
@@ -256,6 +256,21 @@ public with sharing class USER_UserService {
         return ref;
     }
 
+        /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description method to return contact related to current user.
+    */
+    public static Id getContactIdByUserId() {
+        Id userId = UserInfo.getUserId();
+        List<User> contactId = [SELECT ContactId
+                                FROM User
+                                WHERE Id =: userId
+                                WITH SECURITY_ENFORCED
+                                LIMIT 1];
+        return contactId.get(0).ContactId;
```

Comment:
You could do:
User theUser = [SELECT.... WHERE Id =: userId LIMIT 1]

Also contactId is not a proper name as you are getting an unser from the query